package.json

```diff
@@ -5,7 +5,8 @@
     "postinstall": "npm run copy-sjcl",
     "copy-sjcl": "node scripts/generateSJCL.js",
     "bundle-config": "node scripts/bundleConfig.mjs",
-    "build": "npm run build-firefox && npm run build-chrome && npm run build-apple && npm run build-android && npm run build-windows && npm run build-integration && npm run build-chrome-mv3",
+    "build": "npm run build-locales && npm run build-firefox && npm run build-chrome && npm run build-apple && npm run build-android && npm run build-windows && npm run build-integration && npm run build-chrome-mv3",
+    "build-locales": "mkdir -p build/locales && node scripts/buildLocales src/locales/click-to-play > build/locales/ctl-locales.js",
```

Comment:
I figure this command shouldn't need to hard-code "click-to-play". Couldn't it automatically generate the file for each feature that has translated strings?

Reply:
That seems a bit like premature optimization, since I don't know the file / directory structure of upcoming features.  Its easy enough to append another '&& buildLocales' for now, and revise if it becomes an issue?

Reply:
Well, up to you. I just figure there's no need to hard code the names here, we could have the script list the directories in src/locales and produce a bundle for each one. E.g. since src/locales/click-to-play exists, we produce build/locales/click-to-play-locales.js